app/controllers/subsidy/measure-offer.js

```diff
@@ -1,7 +1,63 @@
 import Controller from '@ember/controller';
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+import moment from 'moment';
+import { inject as service } from '@ember/service';
 
 export default class SubsidyMeasureOfferController extends Controller {
-  sort = '-modified';
-  page = 0;
-  size = 20;
+  @service router;
+  @tracked selectedId = null;
+  @tracked selected = null;
+  @tracked options = [];
+
+  async setup() {
+    await this.loadOptions();
+  }
+
+  @action
+  async loadOptions() {
+    this.options = await this.model.series;
+    await this.latestSerie();
+  }
+
+  @action
+  updateSelection(option) {
+    this.selected = option;
+    this.selectedId = option.id;
+
+    this.transitionToSteps();
+  }
+
+  @action
+  routeBack() {
+    this.router.transitionTo('subsidy.index');
+  }
+
+  @action
+  transitionToSteps() {
+    this.router.transitionTo('subsidy.measure-offer.steps', {
+      queryParams: { selectedId: this.selectedId },
```

Comment:
```suggestion
      queryParams: { selectedId: this.selected.id },
```
Bit redundant to create a selectedId in this case